### PR TITLE
"final" PR for Track1

### PR DESCRIPTION
**Pull request by annieengmark**
"final"(?!) PR for Track1 edited content for Q2 workshop. All six pages have been updated in some way. Links are working; missing images still.

### CODE REVIEW

File: Pages/Page 4_ Change code.md [hunk 1]
```diff
@@ -1,32 +1,27 @@
+# Chapter 4: Branches, Merges, Commits
 
-### Readme
-
-Let's start with something small and fix the readme.
+In this section, we will alter the code. We will start with something small and fix the `readme.md.` Before we do that, we want to branch the code to ensure we can track the changes we make. Branching the code is a version control method; we keep the original files on the master branch and create a separate branch where we will make the changes. When we are confident our changes are correct, we will merge the new branch to the existing master brach. The updated files on the new branch essentially overwrite the files on the master branch.
```
Comment (crkoehnen) [inline Pages/Page 4_ Change code.md:3]: I really like this.

**Comment by crkoehnen** [comment]
This looks good to me!

**Comment by brian-dawn** [comment]
A lot of good stuff, nothing stands out as needing to be changed. Awesome work! :D

File: Pages/Page 4_ Change code.md (hunk 1 above)
Comment (mulchy) [inline Pages/Page 4_ Change code.md:3]: +1

File: Pages/Page 1_Intro TOC.md [hunk 2]
```diff
@@ -46,23 +42,45 @@ _insert image of app_
 * Firefox with Firebug
 * The Heroku Toolbelt
 
-Everything should be set up the night before during our install-fest.
-Please ensure you have everything working _before_ you show up for
-ClojureBridge on Saturday.
+_These should have all been installed at the Friday night Install Fest_
+
+
+## Readme
```
Comment (mulchy) [inline Pages/Page 1_Intro TOC.md:48]: I think this Readme section could be removed if you felt like it.

**Comment by mulchy** [comment]
Sorry I didn't get a chance to review this last night, and now I see that it is already merged. Just wanted to say it looks great and is MUCH easier to understand and follow!

**Pull request merged**